controller/contour/daemonset.go

```diff
@@ -202,28 +189,44 @@ func DesiredDaemonSet(contour *operatorv1alpha1.Contour, contourImage, envoyImag
 					},
 				},
 			},
+			ReadinessProbe: &corev1.Probe{
+				FailureThreshold: int32(3),
+				Handler: corev1.Handler{
+					HTTPGet: &corev1.HTTPGetAction{
+						Scheme: corev1.URISchemeHTTP,
+						Path:   "/ready",
+						Port:   intstr.IntOrString{IntVal: int32(8002)},
+					},
+				},
+				InitialDelaySeconds: int32(3),
+				PeriodSeconds:       int32(4),
+				SuccessThreshold:    int32(1),
+				TimeoutSeconds:      int32(1),
+			},
 			Ports: []corev1.ContainerPort{
 				{
 					Name:          "http",
-					ContainerPort: 80,
+					ContainerPort: int32(httpPort),
+					HostPort:      int32(httpPort),
```

Comment:
Do we need host ports?  Seems like #52 (which adds a "LoadBalancer"-type service) obviates need for host ports or host networking.

Reply:
I don't think host ports are necessary, but they are specified [upstream](https://github.com/projectcontour/contour/blob/main/examples/contour/03-envoy.yaml) so I prefer to keep the definitions for the time being. I created https://github.com/projectcontour/contour-operator/issues/70 to address network endpoint publishing. Also see https://kubernetes.slack.com/archives/C8XRH2R4J/p1603393745340600.

Reply:
@Miciah I changed my position in the above comment. I'm removing `hostPort` definitions and will add `externalPolicy: Local` to https://github.com/projectcontour/contour-operator/pull/52.

Reply:
@Miciah and @jpeach I have changed my position on this issue yet again. Please see the godoc reference for `ContainerPort`. This issue will ultimately be resolved in https://github.com/projectcontour/contour-operator/issues/70.